class-pinterest-for-woocommerce.php

```diff
@@ -295,6 +295,31 @@ public function init_plugin() {
 
 		}
 
+		/**
+		 * Initialise Tracker and add trackers to it.
+		 *
+		 * @since x.x.x
+		 *
+		 * @return Pinterest\Tracking|false
+		 */
+		public static function init_tracking() {
+			$is_tracking_enabled             = ! apply_filters( 'woocommerce_pinterest_disable_tracking', false );
+			$is_tracking_conversions_enabled = Pinterest_For_Woocommerce()::get_setting( 'track_conversions' );
+			$is_tracked_site                 = ! wp_doing_cron() && ! is_admin();
+
+			if ( ! $is_tracking_enabled || ! $is_tracking_conversions_enabled || ! $is_tracked_site ) {
+				return false;
+			}
+
+			$tracking = new Pinterest\Tracking();
```

Comment:
Can we at least get Tracking into scope? Consider pulling everything directly if it makes sense.

`new User(...` reads much better.